.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+.DS_Store
```

Comment:
Please remove `.DS_Store` from commit. It shouldn't be pushed to git.

Reply:
Why did we add it in the last commit "Please add .DS_Store to .gitignore"? What is .DS_Store?

Reply:
In the previous commit I asked you to add this file to `.gitignore`.
It shouldn't be pushed to our project in git.
Now I ask you to remove already tracked file from from commit.